package.json

```diff
@@ -29,13 +28,13 @@
     "dateformat": "^3.0.3",
     "font-awesome": "^4.7.0",
     "jquery": "^3.3.1",
-    "moment": "^2.22.2",
```

Comment:
where we are using this package??

Reply:
@inyee786 
Here `moment` using in calcuate master build count 
**Moment package :**
> Parse, validate, manipulate, and display dates and times in JavaScript